src/components/table/table.scss

```diff
@@ -130,6 +130,10 @@ $table--limel-table--row-selector: 1;
             justify-content: center;
         }
     }
+
+    .tabulator-col-resize-handle {
+        display: none;
+    }
```

Comment:
removes this annoying effect:
https://github.com/Lundalogik/lime-elements/assets/35954987/16317cbb-c59d-4653-a954-0c10c7526f96